symphony/app/fbcnms-projects/workflows-proxy/src/proxy/utils.js

```diff
@@ -42,7 +42,6 @@ const SYSTEM_TASK_TYPES: Array<string> = [
   'EXCLUSIVE_JOIN',
   'WAIT',
   'DYNAMIC',
-  'LAMBDA',
```

Comment:
Are the wasm workers ready enough that we can disable lambda right away?

Reply:
It is up to discussion if we want to do it now, but there is no way to fix the security problems associated with lambdas.